### ISSUE TITLE
Add a "fetch only" option

### ISSUE THREAD

**Issue by MisterFruits** [easybuilders/easybuild-framework#2456]
There is currently a way to easily get tarballs for an easyconfig (and dependencies). This is very convenient when the WWW is not accessible from where installation takes place.

It can be done this way:

```` shell
eb --stop fetch --ignore-osdeps --robot <easyconfig.eb>
````

Nevertheless it still requires a module tool on the machine. While it could be easy to install, it also could no be wanted.

I which I could work on this issue if implementation is not too tricky. I had some experimentation and there is an easy way to do that: create a  modulestool that mocks a modules tools (eg the one in the regression test echoing instead of passing modules command).

I ended up with the following command downloading all the tarball I needed without have any modulestool installed:

``` shell
eb --stop fetch --ignore-osdeps --robot --modules-tool=MockModulesTool --module-syntax=Tcl <easyconfig.eb>
````

There is in my opinion 2 big drawbacks to this implementation:
 - when used with something else than ``--stop fetch``, ``--modules-tool=MockModulesTool`` will make the build fail in a non obvious/explicit way
 - people knowing easybuild would eventually figure out all this option while a beginner will have to ask. Also the ``--module-syntax=Tcl`` has no sense.

What do you think ?

Maybe creating a ``--fetch`` option implying ``--stop fetch --ignore-osdeps --modules-tool=MockModulesTool --module-syntax=Tcl`` and making ``MockModulesTool`` not eventually 'callable' would be the solution ?

**Comment by MisterFruits** [comment]
I started implementing this feature. Few interrogation:
 - would ``--fetch`` belongs to "Basic Options" ?
 - where should I document the feature in the documentation ? (beside the option page)

**Comment by Flamefire** [comment]
Closed by #2457

BTW: Can this be used to fetch ALL given sources no matter if their dependencies have been installed already?

**Comment by ocaisa** [comment]
@Flamefire Yes, if you couple it with `--robot`.
